docs/sources/alerting/set-up/provision-alerting-resources/terraform-provisioning/index.md

```diff
@@ -155,209 +156,228 @@ EOT
-
-```HCL
-resource "grafana_notification_policy" "my_policy" {
-    group_by = ["alertname"]
-    contact_point = grafana_contact_point.my_slack_contact_point.name
-
-    group_wait = "45s"
-    group_interval = "6m"
-    repeat_interval = "3h"
-
-    policy {
-        matcher {
-            label = "a"
-            match = "="
-            value = "b"
-        }
-        group_by = ["..."]
-        contact_point = grafana_contact_point.a_different_contact_point.name
-        mute_timings = [grafana_mute_timing.my_mute_timing.name]
-
-        policy {
-            matcher {
-                label = "sublabel"
-                match = "="
-                value = "subvalue"
-            }
-            contact_point = grafana_contact_point.a_third_contact_point.name
-            group_by = ["..."]
-        }
-    }
-}
-```
+To provision notification policies and routing, refer to the [grafana_notification_policy schema](https://registry.terraform.io/providers/grafana/grafana/latest/docs/resources/notification_policy), and complete the following steps.
+
+1. Copy this code block into a `.tf` file on your local machine.
+
+   In this example, the alerts are grouped by `alertname`, which means that any notifications coming from alerts which share the same name, are grouped into the same Slack message. You can provide any set of label keys here, or you can use the special label `"..."` to route by all label keys, sending each alert in a separate notification.
+
+   If you want to route specific notifications differently, you can add sub-policies. Sub-policies allow you to apply routing to different alerts based on label matching. In this example, we apply a mute timing to all alerts with the label a=b.
+
+   ```HCL
+   resource "grafana_notification_policy" "my_policy" {
+       group_by = ["alertname"]
+       contact_point = grafana_contact_point.my_slack_contact_point.name
+
+       group_wait = "45s"
+       group_interval = "6m"
+       repeat_interval = "3h"
+
+       policy {
+           matcher {
+               label = "a"
+               match = "="
+               value = "b"
+           }
+           group_by = ["..."]
+           contact_point = grafana_contact_point.a_different_contact_point.name
+           mute_timings = [grafana_mute_timing.my_mute_timing.name]
+
+           policy {
+               matcher {
+                   label = "sublabel"
+                   match = "="
+                   value = "subvalue"
+               }
+               contact_point = grafana_contact_point.a_third_contact_point.name
+               group_by = ["..."]
+           }
+       }
+   }
+   ```
 
 2. In the mute_timings field, link a mute timing to your notification policy.
 
-3. Run the command ‘terraform apply’.
+3. Run the command `terraform apply`.
 
 4. Go to the Grafana UI and check the details of your notification policy.
 
-**Note:**
+   **Note:**
```

Comment:
Use admonition for note {{< admonition type="note" >}}
{{< /admonition >}}

Reply:
and lazy numbering for all please:)

Reply:
fixed by https://github.com/grafana/grafana/pull/82221/commits/abbd39ca219b9e8d137ae46320a4fc0ee7c15456